### DISCUSSION TITLE
[gatsby-plugin-force-trailing-slashes] New trailingSlash does not provide all options

### DISCUSSION THREAD

**Discussion by rwieruch** [gatsbyjs/gatsby#35680] [Help]
I used the plugin previously this way:

```
    {
      resolve: 'gatsby-plugin-force-trailing-slashes',
      options: {
        excludedPaths: ['/404.html'],
      },
    },
```

Now when using just `trailingSlash: 'always',`, I get paths like `/rss.xml/` for example. How can I avoid this with the new built-in option?

**Comment by LekoArts** [comment]
Are you linking to this via a `gatsby-link`? We only change links when gatsby-link is used.

Anyways, I'll just add it to this check here: https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-page-utils/src/apply-trailing-slash-option.ts#L8

**Reply by LekoArts** [reply]
I've put up https://github.com/gatsbyjs/gatsby/pull/35681 now

**Reply by rwieruch** [reply]
I am using `import { Link as GatsbyLink } from 'gatsby';`

Regarding XML files, I display a link to my RSS feed (see [navigation](https://www.robinwieruch.de/)) and internall to my sitemap.xml FWIW.

Thanks for the quick turnaround here! Super surpised! 🎉

**Reply by LekoArts** [reply]
Okay, yeah, then that's what I expected. _Actually_, you shouldn't use `<Link />` for it but a normal anchor tag as it's not an internal page but an external file. But the PR will just handle the most common assets then.
